app/models/ability.rb

```diff
@@ -50,7 +50,12 @@ def not_signed_in
         registration.new_record?
       end
 
-      can [:new, :create], Registration do |registration|
+      can [:new], Registration do |registration|
+        conf = registration.conference
+        conf.registration_open? && registration.new_record? && !conf.registration_limit_exceeded?
+      end
+
+      can [:create], Registration do |registration|
```

Comment:
We do not want to allow `create` action either, when the registration_limit is reached.